### ISSUE TITLE
Code 1 Error

### ISSUE THREAD

**Issue by John88Texas** [ZencashOfficial/zencash-swing-wallet-ui#78]
I am getting the code 1 issue listed here:

https://github.com/ZencashOfficial/zencash-swing-wallet-ui/blob/master/docs/TroubleshootingGuide.md

I have backed up my wallet.dat, downloaded the new ZENCash Desktop GUI Wallet for Windows 0.75.8a and placed the wallet.dat into the EXE file and tried to run it. I am getting the same error and cannot re-download the blockchain.

I am running Windows 10

**Comment by John88Texas** [comment]
I re downloaded  ZENCash Desktop GUI Wallet for Windows 0.75.8a and tried running without placing the old wallet.dat file in it and it still will not start.

**Comment by John88Texas** [comment]
[debug.log](https://github.com/ZencashOfficial/zencash-swing-wallet-ui/files/1793803/debug.log)

**Comment by vaklinov** [comment]
The essential error in the log is:
```
2018-03-08 15:51:31 ERROR: LoadBlockIndex(): CheckProofOfWork failed: CBlockIndex(pprev=0x94711a0, nHeight=263054, merkle=06b4cd02948796d9a938321c5975d1e8cc435ffdc2c2748a125395d7414707d8, hashBlock=2e213bd52150ed84270024ce8d5259e2fd5fce09701e9ba969cca880138242d1)
```
This means that you have the known problem of a broken local block-chain. In some cases running the latest version of the wallet: ZENCash Desktop GUI Wallet for Windows 0.75.8a, is not sufficient to solve the problem. You also need to manually reindex the block-chain.

The following step-by-step guide for Windows users (created by a third party) is also handy: https://miningwithnvidia.com/2018/02/09/zencash-wallet-eof-reached-code-1/ to help with reindexing in case of encountering the common block-chain problem.

As for the `wallet dat`: It needs to be in directory `%APPDATA%/Zen/` - not "placed ... into the EXE".

**Comment by John88Texas** [comment]
I was able to re-sync up to 270,000 blocks (4 days) then my computer rebooted. Now When I attempt to open the wallet I get this error (attached)
![zen error](https://user-images.githubusercontent.com/37185762/37376970-19aade48-26f5-11e8-86f1-5eced544b4d6.PNG)

**Comment by vaklinov** [comment]
The error is strange. Could you please check that the file `zen-cli.exe`:
1. Exists and has the same content as the one from the wallet ZIP you downloaded.
2. Is executable by your Windows user account - you may try to run it from a command prompt like `zen-cli getinfo`

**Comment by John88Texas** [comment]
I loaded a copy of the blockchain at 200k blocks then accessed my wallet. I could not get the errors to stop but in between them I was able remove my coins from the wallet. We can close this ticket, I have deleted the wallet.
Suggestions.
1. In the future would it be possible for you to make a wallet that does not require the entire blockchain to be downloaded? Since we are not POS or POW with the wallet it should be doable.
2. Holding the coin long term would be better encouraged if POS or mining were more advantageous if holding say 1k coins. Maybe one rate for 49 coins and a different rate for 1,000 coins.  I think your goal was to encourage as many nodes as possible and I like that, I (and others) are just not willing to make 500 nodes to get passive income from my investment. It's just easier to move the money into VEN or WTC where they will have bonuses for holding more tokens.
No need to reply, I know you get hit with ideas from all sides. Best wishes.
